src/conditions/BUILD

```diff
@@ -167,6 +167,43 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "windows_x86_64",
+    constraint_values = [
+        "@platforms//os:windows",
+        "@platforms//cpu:x86_64",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "windows_arm64_constraint",
+    constraint_values = [
+        "@platforms//os:windows",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+config_setting(
```

Comment:
I think we won't need them after https://github.com/bazelbuild/bazel/issues/7260 is fixed, can you put a TODO here?

@oquenchil Can you confirm this?

Reply:
I don't know much about #7260, about what's needed or not. Can you clarify why these config settings won't be needed?

Reply:
If toolchain resolution for C++ is enabled, we no longer need to use `--cpu=x64_windows` to select toolchain, then this config setting won't be needed.